### PR TITLE
Duo/fix tpv6

### PR DESCRIPTION
**Pull request by daisy20170101**
I realized some small errors that have lead to the problem in tpv6. Now i have a proper rupture.

### CODE REVIEW

**Comment by daisy20170101** [comment]
related to 
https://github.com/SeisSol/SeisSol/discussions/590

**Review by sebwolf-de — APPROVED** [review]
LGTM

**Comment by ravil-mobile** [comment]
@sebwolf-de , are you going to integrate it to `dr/cpp`?

**Comment by sebwolf-de** [comment]
Yes

**Comment by Thomas-Ulrich** [comment]
The strength should be - mud *min(0, tn0), as in other friction laws not mud* abs(tn0) 

Le 1 juillet 2022 14:10:28 GMT+02:00, Sebastian Wolf ***@***.***> a écrit :
***@***.*** approved this pull request.

File: src/Physics/ini_model_DR.f90 [hunk 1]
```diff
@@ -734,7 +734,7 @@ SUBROUTINE friction_LSW6(DISC,EQN,MESH,BND)
   EQN%IniMu(:,:) = DISC%DynRup%Mu_S(:,:)
   do iFace = 1, MESH%Fault%nSide
     do iBndGP = 1,DISC%Galerkin%nBndGP
-      DISC%DynRup%Strength(iBndGP,iFace) = EQN%IniMu(iBndGP,iFace) * EQN%InitialStressInFaultCS(iBndGP,1,iFace)
+      DISC%DynRup%Strength(iBndGP,iFace) = EQN%IniMu(iBndGP,iFace) * abs(EQN%InitialStressInFaultCS(iBndGP,1,iFace))
```
Comment (Thomas-Ulrich) [inline src/Physics/ini_model_DR.f90:737]: Should it be:
```
Strength = -DISC%DynRup%cohesion(:,iFace) - DISC%DynRup%Mu(:,iFace) * MIN(EQN%InitialStressInFaultCS(iBndGP,1,iFace),ZERO)  
```?

**Review by Thomas-Ulrich — CHANGES_REQUESTED** [review]
see comment

**Review by Thomas-Ulrich — APPROVED** [review]

File: src/Physics/ini_model_DR.f90 (hunk 1 above)
Comment (daisy20170101) [reply src/Physics/ini_model_DR.f90]: Thanks Thomas!  how about now?

**Pull request merged**